backend/server.py

```
@@ -4595,6 +4603,28 @@ async def list_skills(
     return {"skills": [s.as_dict() for s in skills[:limit]], "total": len(skills)}
 
 
+@app.get("/api/skills/discover")
+async def discover_remote_skills(user: dict = Depends(get_current_user)):
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)